From 0a9941391a9090e54a9e9613c5cb1febf02b85fc Mon Sep 17 00:00:00 2001
From: Alexei Cazacov <alexei.cazacov@qt.io>
Date: Wed, 30 Jul 2025 12:53:43 +0300
Subject: [PATCH 107/239] Doc: add links to the QQuickWidget examples to the
 submodule page

This commit adds links to the QQuickWidget examples to the
Qt Quick Widgets page.

Task-number: QTBUG-134102
Pick-to: 6.9 6.8
Change-Id: If63d106de76a3150025d8a5bfb91007a15b1d02f
Reviewed-by: Andreas Eliasson <andreas.eliasson@qt.io>
(cherry picked from commit 6fb81c0e51a496878e606d3888e8f9525d18b92b)
Reviewed-by: Qt Cherry-pick Bot <cherrypick_bot@qt-project.org>
---
 src/quickwidgets/doc/src/qtquickwidgets-index.qdoc | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/src/quickwidgets/doc/src/qtquickwidgets-index.qdoc b/src/quickwidgets/doc/src/qtquickwidgets-index.qdoc
index b32c40382d..05073e8f50 100644
--- a/src/quickwidgets/doc/src/qtquickwidgets-index.qdoc
+++ b/src/quickwidgets/doc/src/qtquickwidgets-index.qdoc
@@ -45,6 +45,15 @@
         \li \l {Qt Quick Widgets C++ Classes}
     \endlist
 
+    \section1 Examples
+
+    \list
+        \li \l {Qt Quick Widgets Example}
+        \li \l {QQuickWidget - QQuickView Comparison Example}
+        \li \l {QML Previewer Example}
+    \endlist
+
+
     \section1 Licenses and Attributions
 
     Qt Quick Widget is available under commercial licenses from
-- 
2.51.2

